src/lib/vault/reportProblemWithFormSubmission.ts

```diff
@@ -30,12 +35,12 @@ export async function reportProblemWithFormSubmission(
           ":status": "Problem",
           ":problemTimestamp": Date.now(),
         },
-        ReturnValuesOnConditionCheckFailure: "NONE",
+        ReturnValuesOnConditionCheckFailure: "ALL_OLD",
```

Comment:
Ah this is interesting, I didn't know about this feature of DynamoDB conditional writes.